common/test/src/ui/tst_MapDocument.cpp

```diff
@@ -462,6 +463,73 @@ TEST_CASE_METHOD(MapDocumentTest, "canUpdateLinkedGroups")
     document->canUpdateLinkedGroups(kdl::vec_static_cast<mdl::Node*>(entityNodes)));
 }
 
+TEST_CASE_METHOD(MapDocumentTest, "newDocument")
```

Comment:
Sorry I didn't notice this before, but you don't need to use `MapDocumentTest` because you don't need test fixtures. A simple `TEST_CASE` is enough here.

Reply:
Done.